test/system/mangas_test.rb

```diff
@@ -0,0 +1,41 @@
+require "application_system_test_case"
+
+class MangasTest < ApplicationSystemTestCase
+  setup do
+    @manga = mangas(:one)
+  end
+
+  test "visiting the index" do
+    visit mangas_url
+    assert_selector "h1", text: "Mangas"
+  end
+
+  test "creating a Manga" do
+    visit mangas_url
+    click_on "New Manga"
+
+    click_on "Create Manga"
+
+    assert_text "Manga was successfully created"
+    click_on "Back"
```

Comment:
I think it's not necessary to use ' click on "Back" 'for this test to run, you can probably remove it